### ISSUE TITLE
vscode statusbar 

### ISSUE THREAD

**Issue by HUAHUAI23** [ntwigs/Magoon#23]
OS：windows11
VScode：Version: 1.66.2 (user setup)
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:46:01.075Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Windows_NT x64 10.0.22000

statusbar and menu colorscheme seem to be not working
![image](https://user-images.githubusercontent.com/43649186/166208713-45b8ce7f-54d8-46a5-9def-ee4f8c30afcc.png)

**Comment by ntwigs** [comment]
Yup, getting the same menu over here. That's going to need some fixing 😬 However, I'm not getting that purple status bar with the exact same setup. Are you getting the correct status bar color when using other themes? @HUAHUAI23

**Comment by HUAHUAI23** [comment]
sorry for the late reply, i get correct status bar color when using other themes, i guess it might be because of the different operating systems

**Comment by ntwigs** [comment]
@HUAHUAI23 Hmm.. weird.. it seems to be working on w11 for me 🤔 I might get some time this weekend to try and fix this! 😁 Wish me luck 🪄

**Comment by HUAHUAI23** [comment]
haha .. many thanks, i plan to adapt this colorscheme to neovim, i have now moved from vscode to neovim

**Comment by ntwigs** [comment]
That's great! 🙏 Feel free to make a PR when you're done with the nvim port! 😁

**Comment by ntwigs** [comment]
Figured this one out, turns out there's a different bar selector if you open a single file with `vscode` vs if you open a folder.

**Comment by HUAHUAI23** [comment]
nice, so if i open a folder, it will ok

**Comment by ntwigs** [comment]
Should be! Still need to fix that though, so I I'm keeping it open 🙏
